### ISSUE TITLE
GINE: Differences in implementation to the original

### ISSUE THREAD

**Issue by cluel01** [rusty1s/pytorch_geometric#1311]
## ❓ Questions & Help

Hello together,

currently I'm testing the newly published GINEConv. When comparing the integration of the edge features of the PyG variant with the original on Github (https://github.com/snap-stanford/pretrain-gnns/blob/master/chem/model.py and https://github.com/snap-stanford/pretrain-gnns/blob/master/bio/model.py), I had to realize that your implementation is slightly diverging from the ones in the original. 
In PyG the message function is containing a ReLU:
`    def message(self, x_j, edge_attr):
        return F.relu(x_j + edge_attr)`
While in the original it is just:
`    def message(self, x_j, edge_attr):
        return x_j + edge_attr`
respectively:
`    def message(self, x_j, edge_attr):
        return torch.cat([x_j, edge_attr], dim = 1)`
Is there a reason behind the ReLU ?

Best Regards

**Comment by rusty1s** [comment]
The reason is that is works slightly better :) We follow the [OGB implementation](https://github.com/snap-stanford/ogb/blob/master/examples/graphproppred/mol/conv.py#L31) here, which is implemented by the authors of the pre-training paper themselves.

**Comment by cluel01** [comment]
Thanks :)
